src/locales/i18n.ts

```
@@ -0,0 +1,27 @@
+i18n
+  .use(initReactI18next)
+  .init({
+    resources,
+    lng: 'en', // Default language
+    fallbackLng: 'en',
+    interpolation: {
+      escapeValue: false, // React handles escaping
+    },
+  });
+
+i18n.on("languageChanged", (lang) => document.documentElement.lang = lang)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)